faqs/how-do-i-get-involved.md

```diff
@@ -0,0 +1,10 @@
+---
+title: How can I get involved
```

Comment:
Please format as a question - and perhaps this could be more targeted at devs - something like "How can I try out the Ethereum 2.0 chain?"